packages/editor/types.ts

```diff
@@ -1,9 +1,8 @@
 import type { Theme } from '@joplin/lib/themes/type';
 import type { EditorEvent } from './events';
 
-// Editor commands. For compatibility, the string values of these commands
-// should correspond with the CodeMirror 5 commands:
-// https://codemirror.net/5/doc/manual.html#commands
```

Comment:
The goal of this comment was to help increase plugin compatibility with the CodeMirror 6 editor. For example, the numbered list command was named `textNumberedList` instead of `addOrdererdList`. However, this comment can be confusing and doesn't necessarily apply to new code.